evap/staff/importers.py

```diff
@@ -717,16 +707,15 @@ def save_users_to_db(self):
                     )
                     raise
         if len(created_users) == 0:
-            msg = format_html(_("No users were created."))
+            msg = _("No users were created.")
         else:
             msg = format_html(
-                ngettext(
-                    "Successfully created {user_count} user:",
-                    "Successfully created {user_count} users:",
-                    len(created_users),
-                ).format(user_count=len(created_users))
+                _("Successfully created {object}: {list}"),
```

Comment:
Ah, sorry, this doesn't work as I thought: In German, the "created" also needs to know about pluralization ("wurde erfolgreich erstellt" vs. "wurden ..."), so the whole sentence before the ":" should use `ngettext`